### PR TITLE
Remove backpressure from benchmarks app

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

**Comment by davidfowl** [comment]
I dunno how I feel about this yet.

**Comment by BrennanConroy** [comment]
This isn't for rc1

**Comment by BrennanConroy** [comment]
Lets hear your thoughts

File: benchmarkapps/BenchmarkServer/Startup.cs [hunk 1]
```diff
@@ -12,15 +12,23 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddSignalR()
+            services.AddSignalR(o =>
+            {
```
Comment (davidfowl) [inline benchmarkapps/BenchmarkServer/Startup.cs]: fix the formatting

**Pull request merged**